README.md

```diff
@@ -1 +1,14 @@
-# Dx_BAF
\ No newline at end of file
+# Dx_BAF
+
+All script are made tested using Python 3.6.8
+
```

Comment:
I think it would be better to have a short explanation about the function of this pipeline (if I did not miss it of course :) ). right?